filament/src/RenderPass.h

```diff
@@ -257,16 +261,13 @@ class RenderPass {
         bool hasMorphing : 1;                                           //              1 bit
         bool hasHybridInstancing : 1;                                   //              1 bit
 
-        uint64_t rfu[2];                                                // 16 bytes
-
-        static const uint16_t USER_INSTANCE_MASK = 0x8000u;
-        static const uint16_t INSTANCE_COUNT_MASK = 0x7fffu;
+        uint32_t rfu[3];                                                // 16 bytes
```

Comment:
```suggestion
        uint32_t rfu[3];                                                // 12 bytes
```

Reply:
good catch, I'll fix in another CL since it's just a comment. thanks!